app/src/main/java/org/simple/clinic/instantsearch/InstantSearchUpdate.kt

```diff
@@ -41,7 +41,7 @@ class InstantSearchUpdate : Update<InstantSearchModel, InstantSearchEvent, Insta
         dispatch(HideNoPatientsInFacility, HideNoSearchResults, SearchWithCriteria(criteria, model.facility!!))
       }
       InstantSearchValidator.Result.LengthTooShort -> noChange()
-      InstantSearchValidator.Result.Empty -> dispatch(LoadAllPatients(model.facility!!))
+      InstantSearchValidator.Result.Empty -> dispatch(HideNoPatientsInFacility, HideNoSearchResults, LoadAllPatients(model.facility!!))
```

Comment:
We can statically import `InstantSearchValidator.Result` here for better readability.